x-pack/plugins/infra/types/eui.d.ts

```diff
@@ -116,6 +116,8 @@ declare module '@elastic/eui' {
     loading?: any;
     hasActions?: any;
     message?: any;
+    rowProps?: any;
```

Comment:
VSCode was complaining about missing props on `EuiInMemoryTable`, it used the definition from here